.github/workflows/tests.yaml

```diff
@@ -108,7 +107,6 @@ jobs:
           php-version: 7.4
           extensions: mbstring, intl, zip
           coverage: xdebug
-          tools: composer:v2
```

Comment:
Is this now the default or is this an unrelated change?

Reply:
Maybe a copy paste from my example :smile: 
I forced it on phpMyAdmin to have more performance and have less time spent installing the composer modules

Reply:
But that's a removal of v2 - so slower.

Reply:
>  The latest stable version of composer is set up by default.
https://github.com/marketplace/actions/setup-php-action#wrench-tools-support

And Composer 2 is in the stable channel for a while. See build output:
https://github.com/php-webdriver/php-webdriver/runs/1910336904?check_suite_focus=true#step:3:12